pkg/crc/constants/constants_linux.go

```diff
@@ -1,6 +1,7 @@
 package constants
 
 const (
-	OcBinaryName = "oc"
-	DefaultOcURL = "https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/linux/oc.tar.gz"
+	OcBinaryName  = "oc"
+	DefaultOcURL  = "https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/linux/oc.tar.gz"
+	DefaultBundle = "crc_libvirt_4.1.0-rc.5.tar.xz"
```

Comment:
This can be updated to `crc_vbox_4.1.0.tar.xz`

Reply:
@anjannath This is for linux :)

I will create a separate PR for the updating the bundle artifacts for `4.1.0`

Reply:
Haha.. I meant the Linux one only! :)